packages/monitoring-system/metrics-dashboard/src/ts/firestore/firestore-config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "apiKey": "AIzaSyCNYD0Pp6wnT36GcdxWkRVE9RTWt_2XfsU",
```

Comment:
This key is not a secret -- it just helps point the client to the right Firestore instance. User validation happens on the Firestore backend.